.github/workflows/generate-main-yml.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+outfile="$1"
+
+cat > $outfile <<EOF
+name: Cypress Tests
+
+on:
+  push:
+    branches-ignore:
+      - "renovate/**"
+
+jobs:
+EOF
+
+function add_workflow() {
+  local dir="$1"
+  local name="$2"
+  echo "Adding workflow $dir ($name)"
+  cat >> $outfile <<EOF
+  ${dir}-install:
```

Comment:
Is there a reason why there are separate `-install` and `-run` jobs instead of a single job per example?